frontend/src/libs/components/modal/modal.tsx

```diff
@@ -0,0 +1,47 @@
+import { createPortal } from 'react-dom';
+
+import { IconColor } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+
+import { Button } from '../components.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+  children: React.ReactNode;
+  isDisplayed: boolean;
+  title: string;
+  onClose: () => void;
+};
+
+const Modal: React.FC<Properties> = ({
+  children,
+  isDisplayed,
+  title,
+  onClose,
+}) => {
+  return createPortal(
+    isDisplayed && (
```

Comment:
I slightly changed common modal according to the @Liza-Veis suggestion, you can check it [here](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/213/files#diff-d46dbe8116df7bd9fa204c2bc382baa2d3aa61e67b81cc025b2740b41ca2d2f9), createPortal isn't required

Reply:
Alright, thank you for pointing that out, let me check it out.

Reply:
Yes, I did pull your changes successfully, thanks sir!